spec/integration/init_web_spec.cr

```diff
@@ -2,72 +2,85 @@ require "../spec_helper"
 
 include ShouldRunSuccessfully
 
-Spec.before_each do
```

Comment:
I believe that `Spec.before_each` isn't scoped, so even though it was added in this file, I'm assuming it's being run on ALL specs for the entire project. (unless that's been fixed in Crystal?)